### PR TITLE
[DOCS-573] Updating flow and screenshots

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?
Updates flow and screenshots for the NPM page to match updates.

### Motivation
PM requests

### Preview link

* https://docs-staging.datadoghq.com/gus/npm-updates/network_performance_monitoring/network_page

### CODE REVIEW

File: content/en/network_performance_monitoring/network_page.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+Unresolved source and destination tags are marked as `N/A`. A traffic source or destination endpoint may be unresolved because:
+
+* The host or container source or destination IPs are not tagged with the source or destination tags used for traffic aggregation.
+* The endpoint is outside of your private network, and accordingly is not tagged by the Datadog Agent.
+* The endpoint is a firewall, service mesh or other entity where a Datadog Agent cannot be installed.
+
+Use the *Show Unresolved Flows* toggle in the upper right corner of the data table to filter out flows with unresolved (`N/A`) sources or destinations.
+
+Select any row from the data table to see associated logs and traces for a given _source_ <=> _destination_ flow:
```
Comment (shlomoartzi) [inline content/en/network_performance_monitoring/network_page.md]:
```suggestion
Select any row from the data table to see associated logs, traces, and processes for a given _source_ <=> _destination_ flow:
```

File: content/en/network_performance_monitoring/network_page.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+#### Network Load
+
+The following network load metrics are available:
+
+| Metric | Description |
+| -------- | ------ |
+| **Volume** | The number of bytes sent or received over a period. Measured in bytes (or orders of magnitude thereof) bidirectional.|
+| **Throughput** | The rate of bytes sent or received over a period. Measured in bytes per second, bidirectional. |
+| **Retransmits** | Retransmits represent detected failures that are retransmitted to ensure delivery. Measured in count of retransmits from the `source`. |
```
Comment (shlomoartzi) [inline content/en/network_performance_monitoring/network_page.md]:
```suggestion
| **Retransmits** | Retransmits represent detected failures that are retransmitted to ensure delivery. Measured in count of retransmitted frames from the `source`. |
```

File: content/en/network_performance_monitoring/network_page.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+#### TCP
+
+TCP is a connection-oriented protocol that guarantees in-order delivery of packets. The following TCP metrics are available:
+
+| Metric | Description |
+| -------- | ------ |
+| **Retransmits** | Retransmits represent detected failures that are retransmitted to ensure delivery. Measured in count of retransmits from the `source`. |
+| **Round-trip Time (RTT)** | Round-trip time is a proxy for latency.|
```
Comment (shlomoartzi) [inline content/en/network_performance_monitoring/network_page.md]:
```suggestion
| **Round-trip Time (RTT)** | Round-trip time is a proxy for latency. Measure as the time between a TCP frame being sent and acknowledged. |
```

File: content/en/network_performance_monitoring/network_page.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+| **RTT Variance** | RTT is a proxy for jitter. |
+
+### DNS Resolution
+
+Starting with Agent 7.17+, using reverse-DNS lookup, the Agent resolves IP’s to human-readable domain names for external and internal traffic. DNS allows you to monitor cloud provider endpoints where a Datadog Agent cannot be installed, such as S3 buckets, application load balancers, and API’s. Unrecognizable domain names such as DGA domains from C&C servers may point to network security threats. **DNS is encoded as a tag in Datadog**, so you can use it in search bar queries and the facet panel to aggregate and filter traffic.
```
Comment (shlomoartzi) [inline content/en/network_performance_monitoring/network_page.md]:
```suggestion
Starting with Agent 7.17+, using deep packet inspection, the Agent resolves IP’s to human-readable domain names for external and internal traffic. DNS allows you to monitor cloud provider endpoints where a Datadog Agent cannot be installed, such as S3 buckets, application load balancers, and API’s. Unrecognizable domain names such as DGA domains from C&C servers may point to network security threats. **DNS is encoded as a tag in Datadog**, so you can use it in search bar queries and the facet panel to aggregate and filter traffic.
```

File: content/en/network_performance_monitoring/network_page.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+
+{{< img src="network_performance_monitoring/network_page/dns_aggregation.png" alt="DNS aggregation" >}}
+
+**Note**: DNS resolution is supported only for agents running on a host.
```
Comment (shlomoartzi) [inline content/en/network_performance_monitoring/network_page.md]: 'Agents running in host mode' is a bit different than 'agents running on a host.' We can either replace this with 'agents running in host mode' of if you think it's clearer, we can change this to 'DNS resolution is currently supported for hosts where the system probe is running on the root network namespace, which is usually caused by running the system-probe in a container without using the host network.'

File: content/en/network_performance_monitoring/network_page.md [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+
+{{< img src="network_performance_monitoring/network_page/flow_table_region_az.png" alt="Flow table filtered"  style="width:80%;">}}
+
+You can set the timeframe over which traffic is aggregated using the time selector at the top right of the page:
+
+{{< img src="network_performance_monitoring/network_page/npm_timeframe.png" alt="Time frame NPM"  style="width:30%;">}}
+
+### Facet Panels
+
+The facet panels mirror the tags in your search bar query. Switch between the facet panels with the _Source_ and _Destination_ tabs on top:
```
Comment (sarina-dd) [inline content/en/network_performance_monitoring/network_page.md]:
```suggestion
Facet panels mirror the tags in your search bar query. Switch between the facet panels with the _Source_ and _Destination_ tabs on top:
```

**Pull request merged**